SAGESharp.Tests/Animations/BKDSerializationTests.cs

```diff
@@ -65,37 +65,37 @@ public void Test_Writing_BKD_File_Successfully(SerializationTestCaseData<BKD> te
                 {
                     // IntegerLength = 522
                     Length = 8.700001f,
-                    Entries = new List<BKDEntry>()
```

Comment:
The tests in these files are also failing, but they are very hard to debug and figure out what is the problem, you may want to ignore the ones for YaML for the ones for binary you may want to see where the output file differs and why